src/core/configuration.ts

```diff
@@ -217,12 +213,22 @@ namespace Configuration {
   }
 
   export interface Recommendations {
+    /**
+     * set object to enable location-specific autocomplete recommendations
+     */
+    location: Configuration.Recommendations.Location;
     productSuggestions: Configuration.Recommendations.ProductSuggestions;
     iNav: Configuration.Recommendations.INav;
     pastPurchases: Configuration.Recommendations.PastPurchases;
   }
 
   export namespace Recommendations {
+    export interface Location {
+      minSize: number;
```

Comment:
I believe @Vic-Dev had a comment about `minSize` in the documentation pull request